presto-main/src/main/java/io/prestosql/execution/CreateViewTask.java

```diff
@@ -88,7 +89,13 @@ public String explain(CreateView statement, List<Expression> parameters)
                 .map(field -> new ViewColumn(field.getName().get(), field.getType()))
                 .collect(toImmutableList());
 
-        String data = codec.toJson(new ViewDefinition(sql, session.getCatalog(), session.getSchema(), columns, Optional.of(session.getUser())));
+        // use DEFINER security by default
+        Optional<String> owner = Optional.of(session.getUser());
+        if (statement.getSecurity().orElse(null) == INVOKER) {
+            owner = Optional.empty();
```

Comment:
Why are we not recording the owner for INVOKER views? That mode does not imply the views are not owned by anyone. 

I think the concept of owner as it appears in this code is no longer relevant or accurate. I suggest you rename the variable and usages to "definer".